arkouda-docker/README.md

```diff
@@ -186,10 +186,48 @@ docker build --build-arg ARKOUDA_DISTRO_NAME=$ARKOUDA_DISTRO_NAME --build-arg AR
 ## Running prometheus-arkouda-exporter
 
 ```
-export EXPORTER_VERSION=v2023.05.05
+export EXPORTER_VERSION=v2023.06.16
 
 docker run -e ARKOUDA_METRICS_SERVICE_NAME=localhost -e ARKOUDA_METRICS_SERVICE_PORT=5556 -e POLLING_INTERVAL_SECONDS=5 -e EXPORT_PORT=5080 -e EXPORT_PORT=5080 -p 5080:5080 bearsrus/prometheus-arkouda-exporter:$EXPORTER_VERSION
 
+# arkouda-smp-developer
```

Comment:
it doesn't look like the code block above ever gets closed, this messes up the formatting of everything that comes after

This can be seen [here](https://github.com/hokiegeek2/arkouda-contrib/tree/100_create_arkouda-developer_docker_image/arkouda-docker#running-prometheus-arkouda-exporter)

````suggestion
```
# arkouda-smp-developer
````

Reply:
fixed!